### PR TITLE
[Document] update doc about presto version support details

### PR DESCRIPTION
**Pull request by ajantha-bhat**
update doc about presto version support details

Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [ ] Any interfaces changed? NA
 
 - [ ] Any backward compatibility impacted? NA
 
 - [ ] Document update required? yes

 - [ ] Testing done. NA       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.  NA

### CODE REVIEW

**Comment by ajantha-bhat** [comment]
@QiangCai , @ravipesala : please check

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/2804/

File: docs/presto-guide.md [hunk 1]
```diff
@@ -27,14 +27,25 @@ This tutorial provides a quick introduction to using current integration/presto
 ## Presto Multinode Cluster Setup for Carbondata
 ### Installing Presto
 
-  1. Download the 0.210 version of Presto using:
+To know about which version of presto is supported by this version of carbon, visit 
+https://github.com/apache/carbondata/blob/master/integration/presto/pom.xml
+and look  for ```<presto.version>```
```
Comment (QiangCai) [inline docs/presto-guide.md]:
```suggestion
and look for ```<presto.version>```
```

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/11063/

File: docs/presto-guide.md [hunk 2]
```diff
@@ -27,14 +27,25 @@ This tutorial provides a quick introduction to using current integration/presto
...
+
+_Example:_ 
+  `<presto.version>0.210</presto.version>`
+This means current version of carbon supports presto 0.210 version.   
+
+_Note:_
+Currently carbondata supports only one version of presto, cannot handle multiple versions at same time. If user wish to use older version of presto, then need to use older version of carbon (other old branches, say branch-1.5 and check the supported presto version in it's pom.xml file in integration/presto/)
+
+  1. Download that  version of Presto (say 0.210) using below command:
```
Comment (qiuchenjian) [inline docs/presto-guide.md]:
```suggestion
  1. Download that version of Presto (say 0.210) using below command:
```

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/3035/

File: docs/presto-guide.md (hunk 2 above)
Comment (ajantha-bhat) [reply docs/presto-guide.md]: done

File: docs/presto-guide.md (hunk 1 above)
Comment (ajantha-bhat) [reply docs/presto-guide.md]: done

**Comment by CarbonDataQA** [comment]
Build Failed with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/3037/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/2805/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/11065/

**Comment by zzcclp** [comment]
suggest : use 0.217 as example in doc, now carbon has upgraded to 0.217.

**Comment by ajantha-bhat** [comment]
@zzcclp : I didn't used 0.217 because , by the time user refer it, again it might be new version !
But still I can update it

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/2810/

**Comment by zzcclp** [comment]
@ajantha-bhat I think it's better use the version which is the same as one in pom as example, if upgrade version again, it needs to update doc too.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/3042/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/11070/

**Comment by zzcclp** [comment]
Is there any suggestion? If no, I will merge this pr tomorrow. @QiangCai @qiuchenjian @ravipesala

**Comment by sraghunandan** [comment]
LGTM

**Comment by zzcclp** [comment]
LGTM. Merged.

**Pull request closed without merging**